components/InfoBox/index.scss

```diff
@@ -0,0 +1,36 @@
+@import '~kununu-theme/scss/base/variables';
+
+.infoBox {
+  position: absolute;
+  display: block;
+  z-index: 100;
+  width: 100vw;
```

Comment:
vw in not supported fully from IE (http://caniuse.com/#search=vw)... does this transpiled?